src/js/core/dom.js

```diff
@@ -843,8 +850,12 @@ function splitNode(point, options) {
     return point.node.splitText(point.offset);
   } else {
     const childNode = point.node.childNodes[point.offset];
+    let childNodes = listNext(childNode);
+    // remove empty nodes
+    childNodes = childNodes.filter(function (element) {return !dom_isEmpty(element);});	
```

Comment:
This is throwing an error, dom_isEmpty is not defined. Not sure if perhaps the function may have been accidently left out, or should be present from one of your other PR's.
![image](https://github.com/summernote/summernote/assets/32765473/be05f769-367f-4c59-98c9-9c3309004803)

Reply:
sorry.. it should be "return !isEmpty(element)"
The function will be compiled to dom_isEmpty but in the original code is must be without the prefix.

Reply:
Oppps

Reply:
Yep, that fixed it. I'll do a quick PR for it.